<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>310</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWebdav Example</string>
  </property>
  <property name="windowIcon">
   <iconset resource="explorer.qrc">
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTreeWidget" name="treeWidget">
      <property name="contextMenuPolicy">
       <enum>Qt::ActionsContextMenu</enum>
      </property>
      <property name="acceptDrops">
       <bool>true</bool>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>true</bool>
      </property>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Size</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Time</string>
       </property>
      </column>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionDownload">
   <property name="text">
    <string>Download</string>
   </property>
   <property name="toolTip">
    <string>Download selected files</string>
   </property>
  </action>
  <action name="actionUpload">
   <property name="text">
    <string>Upload</string>
   </property>
   <property name="toolTip">
    <string>Upload selected files</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="text">
    <string>refresh</string>
   </property>
   <property name="toolTip">
    <string>Refresh this folder</string>
   </property>
  </action>
  <action name="actionCreateDir">
   <property name="text">
    <string>New Directory</string>
   </property>
   <property name="toolTip">
    <string>Create a new directory</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="text">
    <string>Delete</string>
   </property>
   <property name="toolTip">
    <string>Delete selected items</string>
   </property>
  </action>
  <action name="actionGo">
   <property name="text">
    <string>Go</string>
   </property>
   <property name="toolTip">
    <string>Load this url</string>
   </property>
  </action>
  <action name="actionParent">
   <property name="text">
    <string>Parent Directory</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionInformations">
   <property name="text">
    <string>Informations</string>
   </property>
   <property name="toolTip">
    <string>Informations</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="explorer.qrc"/>
 </resources>
 <connections/>
</ui>
